src/components/NFTMedia/NFTMedia.module.scss

```diff
@@ -12,7 +11,7 @@
 .Media {
 	object-fit: contain;
 	max-width: 100%;
-	max-height: 100%;
+	// max-height: 100%;
```

Comment:
Make sure to clean up before submitting, generally I think we shouldn't leave commented-out code in a commit, unless there's a comment explaining why it's commented out

Reply:
Done. Sorry, that one just slipped through.

Reply:
Done. That just one slipped through.